src/components/Layout/Layout.tsx

```diff
@@ -210,6 +211,7 @@ export const Layout = ({
           colorMode={colorMode}
         >
           <IconsProvider icons={defaultIcons}>
+            <Modal isGen1={isGen1} />
```

Comment:
Yes, I wasn't sure where this should really be, but my reasoning is that if you're a keyboard only navigator, and there is this thing blocking part of the page, probably you want to be able to tab to it pretty quickly to close it. Since it's non-modal (the rest of the page can still be interacted with unlike a modal dialog that blocks tabbing to other elements and usually has a backdrop preventing clicking through) that seemed like the best option.

Reply:
Thank you for the explanation, that makes sense